### ISSUE TITLE
Documentation - Update Using Woodwork with Dask and Koalas Guide

### ISSUE THREAD

**Issue by thehomebrewnerd** [alteryx/woodwork#607]
Update to use the new accessor approach: https://woodwork.alteryx.com/en/stable/guides/using_woodwork_with_dask_and_koalas.html

**Comment by thehomebrewnerd** [comment]
Need to include that `make_index` is not supported when using a Koalas dataframe. Users can manually create an index column before initializing Woodwork with this code:
```python
df = df.koalas.attach_id_column('distributed-sequence', 'index_col_name')
```

**Comment by thehomebrewnerd** [comment]
Closed by #701
